### ISSUE TITLE
Rule: Setting name should be capitalized

### ISSUE THREAD

**Issue by bhirsz** [bhirsz/robotframework-robocop#153]
Settings names like Suite Setup or Library are case insensitive. It should be reported as warning if used as not capitalized or upper case:
```
*** Settings ***
Suite Setup  Keyword
suite Setup  Keyword
SUITE SETUP  Keyword
```
